src/Game.js

```diff
@@ -20,24 +23,35 @@ class Game {
     }
   }
 
+  turnChange() {
+    round++;
+  }
+
   handleSelect(element) {
     const [x, y] = parseId(element.id);
-
     if (!this.gameArea[x][y]) {
       return;
     }
-
     this.selectedPiece = this.gameArea[x][y];
+    console.log(this.selectedPiece);
+    if (this.selectedPiece.side === "white" && round % 2 === 0) {
```

Comment:
Jestem zdania że wygodniej by było oprócz round (round zostaje bo może się przydać np. by w interfejsie wypisać ile posunięć już nastąpiło) mieć jeszcze właściwość currentPlayer, która przechowa stringa 'white' lub 'black'.  Wtedy ten check mógłby wyglądać prościej i być zapisany tylko raz:
```suggestion
    if (this.selectedPiece.side === this.currentPlayer) {
```